ch2/procfs_simple_intf/procfs_simple_intf.c

```diff
@@ -384,7 +411,12 @@ static int __init procfs_simple_intf_init(void)
 	return 0;	/* success */
 
  out_fail_3:
+
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 5, 0)
```

Comment:
it's from ver 5.9.0 that kfree_sensitive() replaces kzfree(); pl update, thanks

Reply:
Updated. Thank you very much again for another piece of knowledge!